keymanager/src/policy/cached.rs

```diff
@@ -172,10 +172,12 @@ impl Policy {
     fn load_policy(storage: &dyn KeyValue) -> Option<CachedPolicy> {
         let ciphertext = storage.get(POLICY_STORAGE_KEY.to_vec()).unwrap();
 
-        unseal(Keypolicy::MRENCLAVE, POLICY_SEAL_CONTEXT, &ciphertext).map(|plaintext| {
-            // Deserialization failures are fatal, because it is state corruption.
-            CachedPolicy::parse_raw(&plaintext).expect("failed to deserialize persisted policy")
-        })
+        unseal(Keypolicy::MRENCLAVE, POLICY_SEAL_CONTEXT, &ciphertext)
+            .expect("failed to unseal policy (if running in SGX mode, check if the CPU had changed; if yes, wipe 'worker-local-storage.badger.db' and resync the node)")
```

Comment:
Can we just use `unwrap`?

Reply:
Yes, this panic message is not visible in production mode anyway.